src/components/questionStructure/Section.tsx

```diff
@@ -1,20 +1,21 @@
-import { Flex, Grid, Strong, Text } from '@radix-ui/themes'
+import { Flex, Grid, Text } from '@radix-ui/themes'
+import { last } from 'lodash'
 import React, { FC, ReactNode } from 'react'
 
 import { numberToRoman } from '../../utils/common'
 import Markdown from '../Markdown'
 
 interface SectionProps {
-  sectionId: number
+  sectionId: string
   description?: string
   children: ReactNode
 }
 
 const Section: FC<SectionProps> = ({ sectionId, description, children }) => {
   return (
     <Grid columns="1fr 8fr">
-      <Text>
-        <Strong>{numberToRoman(sectionId)})</Strong>
+      <Text weight="bold" id={`q${sectionId}`}>
```

Comment:
I don't think the q prefix is now required. I'd initially thought there would be anchors to questions i.e q1-1-1 and the equivalent mark in the right hand marking summary which would have anchors of m1-1-1

Not a major issue but that's why I originally suggested "q"

Reply:
I think it still confers a little bit of human readability to the id. Don't have a strong opinion on it, but I don't mind leaving it there.